feature/src/main/java/com/terning/feature/filtering/filteringone/navigation/FilteringOneNavigation.kt

```diff
@@ -25,7 +26,7 @@ fun NavGraphBuilder.filteringOneNavGraph(
 ) {
     composable<FilteringOne> {
         val args = it.toRoute<FilteringOne>()
-        FilteringOneScreen(
+        FilteringOneRoute(
             name = args.name,
             onNextClick = { grade -> navHostController.navigateFilteringTwo(grade) },
             navigateUp = { navHostController.navigateUp() }
```

Comment:
```suggestion
            onNextClick = navHostController::navigateFilteringTwo,
            navigateUp = navHostController::navigateUp
```
이 친구들도 요렇게 바꿀 수 있답니다!